src/api/hooks/useCreateRegistrationFromDoi.ts

```diff
@@ -0,0 +1,20 @@
+import { useMutation } from '@tanstack/react-query';
+import { createRegistrationFromDoi } from '../registrationApi';
+import { setNotification } from '../../redux/notificationSlice';
+import { useDispatch } from 'react-redux';
```

Comment:
Vi ønsker helst å sortere imports alfabetisk, med eksterne pakker først, så interne filer. Dette er satt opp automatisk for vscode i `.vscode/settings.json` (se `source.organizeImports`). Sikkert mulig å gjøre noe lignende i WebStorm også? Mener @joachimjorgensen fant en løsning på det i IntelliJ, som jeg vil tro også kan brukes for WebStorm?

Reply:
Fant bare en halvveis løsning. `Settings` -> `Editor` -> `Code Style` -> `TypeScript`. Den sorterer bare innad i en import, men ikke rekkefølgen på imports. Burde se om vi finner en plugin/linter på rekkefølgen også 😬

Reply:
![image](https://github.com/BIBSYSDEV/NVA-Frontend/assets/1130244/e0369344-60c6-4dc5-92eb-c8dc6aec6a0f)
Med denne snarveien kan man rydde opp i en fil. Det fins en setting for å gjøre dette ved lagring, men jeg har ikke sett den faktisk gjøre noen endring.